src/utils/constants.js

```diff
@@ -23,25 +23,105 @@ export const rightVocabs = [1, 10];
 export const numberField = [1, 255];
 
 export const contributors = {
-  desktop: [
-    { name: "noctera", description: "25.11.2020" },
-    { name: "luwol03", description: "17.03.2021" },
-    { name: "zikowang", description: "16.03.2021" },
+  frontend: [
```

Comment:
you need to also adjust the translations. These keys are basically the keys for `contributors.xxxx` in the translations.